src/CompositeChat.cpp

```diff
@@ -45,7 +45,7 @@ void cCompositeChat::Clear(void)
 
 void cCompositeChat::AddTextPart(const AString & a_Message, const AString & a_Style)
 {
-	m_Parts.push_back(TextPart{{ a_Message, a_Style, {} } });
+	m_Parts.emplace_back(TextPart{{ a_Message, a_Style, {} } });
```

Comment:
I think these should remain push_back. Unless you want to write a constructor for all the parts, then you could use `emplace_back(std::in_place_type<TextPart>, ...)`. The constructor is needed because ref http://stackoverflow.com/questions/8782895/why-doesnt-emplace-back-use-uniform-initialization